### PR TITLE
Changing toggle styles to use flex box

### LINKED ISSUE #4307: Toggle thumb is misaligned when overriding text size
### Bug Report

- __Package version(s)__: 5.64.1
- __Browser and OS versions__: Chrome

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) Yes

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable)

#### Describe the issue:

All of the sizing on the pill and thumb class are done in ems except the borderWidth which is set to 1px (https://github.com/OfficeDev/office-ui-fabric-react/blob/ac43f523749423c2ee3cd9ad1f4ba868612c7241/packages/office-ui-fabric-react/src/components/Toggle/Toggle.classNames.ts)
If the font size is overridden, everything scales except the border, meaning 
the thumb's top style also needs to be overridden to keep it vertically aligned in the center.  It's better if the toggle use flexbox positioning so it doesn't need the fractional ems in the styling that break when the font size is changed.

#### Actual behavior:

Overriding the font size misaligns the toggle thumb.

#### Expected behavior:

The toggle thumb should be vertically aligned regardless of the font size

### If applicable, please provide a codepen repro:
thumboffcenter

### PR DESCRIPTION
**Pull request by jdrush89**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #4307
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Removing the absolute positioning on the toggle thumb and using flexbox positioning instead to keep the toggle aligned correctly for all text sizes.

#### Focus areas to test

Test that toggles render correctly for multiple text sizes

### CODE REVIEW

**Comment by jdrush89** [comment]
@betrue-final-final It looks like screener was not picking up my latest commit for whatever reason.  Now it's complaining that the toggle has changed, though running npm run update-snapshots doesn't update anything at this point.  Any ideas?

**Comment by betrue-final-final** [comment]
Not sure. I'll let the screener guy know.

**Comment by betrue-final-final** [comment]
Did you check your snap tests?

**Comment by betrue-final-final** [comment]
Looks like whatever you did fixed it!

**Comment by jdrush89** [comment]
@phkuo snapshots are updated, let me know if it looks good

File: packages/office-ui-fabric-react/src/components/Toggle/__snapshots__/Toggle.test.tsx.snap [hunk 1]
```diff
@@ -48,17 +48,22 @@ exports[`Toggle renders toggle correctly 1`] = `
...
+            display: flex;
             font-size: 20px;
             height: 1em;
-            line-height: 1em;
             outline: transparent;
+            padding-bottom: 0;
+            padding-left: .2em;
+            padding-right: .2em;
+            padding-top: 0;
             position: relative;
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/Toggle/__snapshots__/Toggle.test.tsx.snap]: hmm I'm surprised this didn't get removed

**Pull request merged**